docs/content/docs/start/react-tutorial.md

```diff
@@ -0,0 +1,295 @@
+---
+title: 'Tutorial: Create a Fluid Framework application with React'
+menuPosition: 4
+---
+
+In this tutorial, you'll learn about using the Fluid Framework by building a simple application that enables every client of the application to change a dynamic time stamp on itself and all other clients almost instantly. You'll also learn how to connect the Fluid data layer with a view layer made in [React](https://reactjs.org/). This animated GIF shows what the application looks like when it is open in four clients.
+
+![Animated GIF showing the application open in four clients](https://user-images.githubusercontent.com/1434956/111496992-faf2dc00-86fd-11eb-815d-5cc539d8f3c8.gif)
+
+{{< callout note >}}
+
+This tutorial assumes that you are familiar with the [Fluid Framework Overview](../overview.md) and that you have completed the [QuickStart](./quick-start.md). You should also be familiar with the basics of [React](https://reactjs.org/), [creating React projects](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app), and [React Hooks](https://reactjs.org/docs/hooks-intro.html).
+
+{{< /callout >}}
+
+## Create the project
+
+1. Open a Command Prompt and navigate to the parent folder where you want to create the project; e.g., `c:\My Fluid Projects`.
+1. Run the following command at the prompt. (Note that the CLI is np**x**, not npm. It was installed when you installed npm.)
+
+    ```dotnetcli
+    npx create-react-app fluid-react-tutorial --use-npm
+    ```
+
+1. The project is created in a subfolder named `fluid-react-tutorial`. Navigate to it with the command `cd fluid-react-tutorial`.
+1. The project uses two Fluid libraries:
+
+    |Library |Description |
+    |---|---|
+    |fluid&#x2011;experimental/fluid&#x2011;framework |Contains the SharedMap [DataObject](https://fluidframework.com/docs/glossary/#dataobject) that synchronizes data across clients. _This object will hold the most recent timestamp update made by any client._|
+    |fluid&#x2011;experimental/frs&#x2011;client&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;   |Defines the connection to a Fluid Relay Service (FRS) server and defines the starting schema for the [container](https://fluidframework.com/docs/concepts/containers-runtime/). In this tutorial, the FRS server will run on localhost.|
```

Comment:
```suggestion
    |fluid&#x2011;experimental/frs&#x2011;client&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;   |Defines the connection to an Azure Fluid Relay service (FRS) server and defines the starting schema for the [Fluid container][]. In this tutorial, we will use a local test server called Tinylicious.|
```

Reply:
I'm going to make you proposed change, but leave out "Azure". Since the code is using frs-client to connect to tinylicious, it is not always used to connect to an **Azure** FRS.